docs/joss_paper/paper.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'A Python astronomy package for HiPS: Hierarchical Progressive Surveys'
+tags:
+- python
+- astronomy
+- science
+- image-viewer
+- image-processing
+
+authors:
+- name: Christoph Deil
+  orcid:
+  affiliation: 1
+
+- name: Thomas Boch
+  orcid:
+  affiliation: 2
+
+- name: Adeel Ahmad
+  orcid: 0000-0001-5035-4497
+  affiliation: 3
+
+affiliations:
+- name: Max Planck Institute for Nuclear Physics, Heidelberg, Germany
+  index: 1
+- name: Centre de Données astronomiques de Strasbourg, Strasbourg, France
```

Comment:
My affiliation is:
Université de Strasbourg, CNRS, Observatoire astronomique de Strasbourg